.github/workflows/stale-issues-pr.yml

```diff
@@ -0,0 +1,46 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Awaiting response issues
+        uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 7
```

Comment:
This seems too low, let's use 14

Reply:
To apply stale label  it's 7 days, to close the issue 7 more days, total 14 days.

Reply:
Still seems too low -- let's go for 14+14